Project.toml

```diff
@@ -0,0 +1,7 @@
+name = "VulkanCore"
+uuid = "2acc6841-08b4-44ed-b4cc-ea5ac90711f2"
+
+[deps]
+Clang = "40e3b903-d033-50b4-a0cc-940c62c95e31"
```

Comment:
Why do you use a Project.toml? It's still not yet recommended ;)

Reply:
Started this a few weeks ago. At that point, this looked like the best way to go about things. The julia docs are in dire need of a "Make your own module - for dummies" section...

What's the best way to get the METADATA_compatible_uuid ("16167f82-ea26-5cba-b1de-ed6fd5e30a11") in there, without `Pkg` trying to kill me?

Reply:
Using a Project.toml is fine these days and actually is easier to work with than Requires. The entire CUDA stack has been using Project.toml since end of August

Reply:
> why this package needs Clang.jl as its dependency?

I had a warning pop up while building VulkanCore on 1.0 that stated it uses Clang but doesn't include it in its dependencies. Not entirely sure what it's used for even.

Reply:
Clang.jl is used for generating bindings from c headers. Since users only need those generated Julia codes, there is no need to mark Clang as a dependency.